tests/unit/lib/ftp-tests.js

```diff
@@ -1,11 +1,104 @@
 const EventEmitter = require("events");
 const Future = require("fluture");
 const {expect} = require("chai");
-const {sendFileViaFtp} = require("../../../lib/ftp.js");
+const {getFileViaFtp, sendFileViaFtp} = require("../../../lib/ftp.js");
 const {fork} = Future;
 const {Readable} = require("stream");
 
 describe("Unit Tests - FTP", function() {
```

Comment:
```suggestion
describe("Unit Tests - ftp.js", function() {
```
overall tests should be named using the filename.